backend/typescript/models/index.ts

```diff
@@ -1,7 +1,7 @@
 import * as path from "path";
 import { Sequelize } from "sequelize-typescript";
 
-export const sequelize = new Sequelize(
+export default sequelize = new Sequelize(
```

Comment:
out of curiosity  - when do you use export default vs const? Which es-lint rule was this

Reply:
it's named export vs default export. 
Since I think we're exporting one thing in the file, the linting recommend using default export so that you can just import using the import default syntax: `import name_of_reference from 'index.ts'`
as opposed to named import which you have to specify the name of what you're importing: `import {sequelize} from 'index.ts`